src/client/components/susceptibility-list/susceptibility-list.vue

```diff
@@ -32,14 +32,21 @@
           v-if="selectedFactorIndex === index"
           class="list-item__settings"
         >
-          <span>{{ factor.title }}</span>
           <weight-factor
             :min="factor.min"
             :max="factor.max"
             :step="factor.step"
             :weight-factor="factor.weightFactor"
             @onChange="(value) => $emit('setWeightFactor', { value, index })"
           />
+          <input-range
+            v-if="factor.classes.length"
```

Comment:
check if classes exists first and then if the length is 4. If its not 4 we dont know what to do with it